z3c/dependencychecker/tests/test_report.py

```diff
@@ -627,3 +629,151 @@ def test_exit_status_set(minimal_structure):
     assert report.exit_status == 0
     report.print_report()
     assert report.exit_status == 1
+
+
+PKG_REQUIRES_LINE = ("requires.txt", "zope.interface")
+ZOPE_MAP_REQUIRES_LINE = ("requires.txt", "Zope")
+MAPPING_LINE = ("pyproject.toml", '[tool.dependencychecker]\nZope = ["zope.interface"]')
+CODE_LINE = ("__init__.py", "import zope.interface.Interface")
```

Comment:
Nice way of testing!

I initially got a bit distracted by the term `_LINE`, but I think that can be fixed with a comment:

```suggestion
# These 'lines' are instructions for the parametrized test below. A filename + the desired contents.
PKG_REQUIRES_LINE = ("requires.txt", "zope.interface")
ZOPE_MAP_REQUIRES_LINE = ("requires.txt", "Zope")
MAPPING_LINE = ("pyproject.toml", '[tool.dependencychecker]\nZope = ["zope.interface"]')
CODE_LINE = ("__init__.py", "import zope.interface.Interface")
```